lib/api/web-element/commands/sendKeys.js

```diff
@@ -22,10 +22,10 @@
  * @instance
  * @syntax browser.element(selector).sendKeys(...keys)
  * @param {...string} keys
- * @see https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebElement.html#sendKeys
```

Comment:
We should keep this selenium link as well under `@see` tag. Although this link is giving 404 right now, it is expected to work later on when Selenium fixes the issue related to API docs.